### DISCUSSION TITLE
How change local_Adde without rebuild all the client?

### DISCUSSION THREAD

**Discussion by spectreDeveloper** [aio-libs/aiohttp#8438] [General]
Hi,
I will try to explain what i want reach and why. 

Actually i create one aiohttp clientSession and pass to a function that made almost 100.000k request http 

I need to change the local_add each time I join in the function and made the request to X website. 

Obviously, I can recreate the aiohttp client in the function each time, but this reduce a lot the performance of the program. 

So, there is a way to set a specific connector each time i made a request without edit or rebuild completely the client passed as args of the function?

thanks

**Comment by Dreamsorcerer** [comment]
Well, I imagine you could create a custom connector class which acts as a meta connector, maybe? The meta class can then create connectors internally as needed.
